Dockerfile

```diff
@@ -1394,6 +1394,88 @@ while echo "Running"; do
 done
 EOF
 
+
```

Comment:
We do not really need to have it in PROD dockerfile. PROD image is generally  (in most cases) installed from installed packages, not from sources - and those packages already have assets compiled. Only for local development, npm step/airflow installation is done from local sources, but this is not something we should do frequently and it does not require caching.

Reply:
Removed from PROD dockerfile. Thanks for the details!